ui.apps/src/main/content/jcr_root/apps/saas-aem-module/shared/filters.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root
+    xmlns:sling="http://sling.apache.org/jcr/sling/1.0"
+    xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    xmlns:nt="http://www.jcp.org/jcr/nt/1.0"
+    jcr:primaryType="nt:unstructured"
+    fieldLabel="Filters"
+    fieldDescription="Only simple filter entries can be configured."
```

Comment:
please replace description with "Filter entries with the "AND" relation can be configured"